### DISCUSSION TITLE
Compile nocore with android headers

### DISCUSSION THREAD

**Discussion by nitanmarcel** [gojue/ecapture#390] [Q&A]
How can I compile this in nocore mode for Android using the kernel headers?

Reason I want to do this is that I'm patching my own 4.14.x kernel, so far opensnoop works perfectly fine but I'm missing the debug bpf info commits to boot recapture to test it.

**Comment by cfc4n** [comment]
as same as linux.

I compiled it using Linux. For an arm64 Android system, you'll need to execute the makefile on a Linux arm64 platform. 

You can refer to the `release script` for guidance.

https://github.com/gojue/ecapture/blob/71558a8a2f8716a388acbdc311c00c00a2c9ac4c/.github/workflows/release.yml#L172-L199

**Reply by nitanmarcel** [reply]
My main question was: __How can I compile this in nocore mode for Android using the kernel headers?__

**Reply by cfc4n** [reply]
modify their values yourself: `KERN_BUILD_PATH` and `KERN_SRC_PATH`  

https://github.com/gojue/ecapture/blob/71558a8a2f8716a388acbdc311c00c00a2c9ac4c/Makefile#L181-L182
